rosbag2_transport/CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package rosbag2_transport
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.15.0 (2022-04-05)
+-------------------
+* support to publish as loaned message (`#981 <https://github.com/ros2/rosbag2/issues/981>`_)
+* Contributors: Barry Xu
```

Comment:
Haven't we missed here 
```diff
+ * Revert "Add the ability to record any key/value pair in the 'custom' field in metadata.yaml (`#976 <https://github.com/ros2/rosbag2/issues/976>`_)" (`#984 <https://github.com/ros2/rosbag2/issues/984>`_)
+ * Add the ability to record any key/value pair in the 'custom' field in metadata.yaml (`#976 <https://github.com/ros2/rosbag2/issues/976>`_)
+ * Contributors: Jorge Perez, Tony Peng
```
As well ?

Reply:
These changes didn't touch `rosbag2_transport`, which is why it is missing from there.  With that (and the approval), I'll go ahead and resolve this conversation.